Amazon_Linux_Base/build.sh

```diff
@@ -6,6 +6,7 @@ IMAGE="amazon-linux2-base"
 
 if [ "${PLATFORM}x" = "arm64x" ]; then
   docker build --platform linux/${PLATFORM} -t "${IMAGE}:arm" . 
+  docker build --platform linux/amd64 -t $IMAGE . 
```

Comment:
The condition says `arm64x` but the flag is `amd64` is this a mistake or am I missing something?

Reply:
I think I got it. We build both base images if the flag is arm

Reply:
In this case we would need to build both versions of the image, as the PNC emulator cannot run on arm64. If we only build amd then all the other images which depend on this one will need a lot more resources to run. Building two images seemed the best of both worlds.